site-src/v1alpha2/guides/tls.md

```diff
@@ -130,28 +130,6 @@ there are two HTTPRoute resources which specify certificates for
 {% include 'tls-cert-in-route.yaml' %}
 ```
 
-## Upstream TLS
-
-Upstream TLS configuration applies to the connection between the Gateway
-and Service.
-
-There is only one way to configure upstream TLS: using the `BackendPolicy`
-resource.
-
-Please note that the TLS configuration is related to the Service or backend
-resource and not related to a specific route resource.
-
-### Example
-
-The following example shows how upstream TLS can be configured. We have
-omitted downstream TLS configuration for simplicity. As noted before, it
-doesn't matter how downstream TLS is configured for the specific listener or
-route.
-
-```yaml
-{% include 'upstream-tls.yaml' %}
```

Comment:
Please delete this file.

Reply:
To clarify, this is just the upstream-tls example right? In that case, it's still used by v1alpha1 docs, and probably will be as long as we keep v1alpha1 docs around.